docs/backend/setup/database-migrations.md

```diff
@@ -98,56 +98,112 @@ When you are writing the code for your new migration, there are a few things you
 
 ### Adding a new table
 
-If you are creating a new table, first generate a new migration file. When creating the SQL you may write the migration like this:
-
-```sql
-create table new_table
-(
-    id uuid
-        constraint new_table_pkey primary key,
-    column1 text not null,
-    column2 text,
-    created_at timestamp not null,
-    updated_at timestamp not null
-);
-```
-
-Then create a new models file in `pkg/models/` named after your new table. For this example, that would be `new_table.go`. The contents will look like:
+This section covers what it's like to add a new table, though some information is helpful when editing existing 
+tables/models as well.
+
+For the examples in this section, we'll be adding a new model called `Cat`, with a corresponding table called `cats`.
+
+:::note Table Name vs Model Name
+
+Notice how the model name is singular, while the table name is pluralized. This is a convention of `Pop`, our ORM.
+See the [conventions Pop follows](https://www.gobuffalo.io/en/docs/db/getting-started) for more info.
+
+:::
+
+1. [Generate a new migration file](#creating-migrations).
+2. Open the generated SQL file and add the logic to create your table.  When creating the SQL you may write the migration like this:
+
+    ```sql
+    CREATE TABLE cats
+    (
+        id uuid PRIMARY KEY NOT NULL,
+        created_at timestamp NOT NULL,
+        updated_at timestamp NOT NULL,
+        name text NOT NULL,
+        birthday date,
+        gotcha_day date,
+        bio text,
+        weight int
+    );
+    
+    COMMENT on TABLE cats IS 'Store cats and their details.';
+    COMMENT on COLUMN cats.name IS 'The official name of a cat...nicknames might need their own table...';
+    COMMENT on COLUMN cats.birthday IS 'Date the cat was born.';
+    COMMENT on COLUMN cats.gotcha_day IS 'Date the cat was adopted. May be same as birthday.';
+    COMMENT on COLUMN cats.bio IS 'Big text field for owners to tell you everything about their cat.';
+    COMMENT on COLUMN cats.weight IS 'Current weight of the cat, or at least what they weighed the last time they let you weigh them.';
+    ```
 
-```go
-package models
-
-import (
-  "time"
-
-  "github.com/gobuffalo/pop"
-  "github.com/gobuffalo/validate"
-  "github.com/gobuffalo/validate/validators"
-  "github.com/gofrs/uuid"
-)
-
-// NewTable represents a new table
-type NewTable struct {
-  ID        uuid.UUID `json:"id" db:"id"`
-  Column1   string    `json:"column1" db:"column1"`
-  Column2   *string   `json:"column2" db:"colunn2"`
-  CreatedAt time.Time `json:"created_at" db:"created_at"`
-  UpdatedAt time.Time `json:"updated_at" db:"updated_at"`
-}
+    Some things to note:
+
+    1. The SQL keywords don't need to be capitalized, but it can be nice to differentiate some of them from certain 
+       things like the field names or types.
+    2. We want to add columns for every column other than the `id`, `created_at`, and `updated_at` fields. This makes it
```

Comment:
I think you meant `We want to add comments for every column...` here. Using `1.` for number lists is the convention as I understand since docusaurus will auto number them anyway.

Reply:
🤦🏼 thanks fro catching that typo

Ugh, yeah I keep having to fight my editor because it's trying to be helpful by auto-updating the numbers for me whenever I add one and I forgot to got back at the end to fix them.

Reply:
I'm only good at catching typos because I've made so many. Sorry your editor is fighting you. I wonder if we could add a linter or something that would help fix it on save or something.